### ISSUE TITLE
Test runner fails

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/scala-test-runner#39]
With the latest [image upgrade](https://github.com/exercism/scala-test-runner/pull/37), we're now running out of memory:

```shell
two-fer: testing...\n[0.020s][warning][os,thread] Failed to start thread \"GC Thread#0\" - pthread_create failed (EPERM) for attributes: stacksize: 1024k, guardsize: 4k, detached.\n#\n# There is insufficient memory for the Java Runtime Environment to continue.\n# Cannot create worker GC thread. Out of system resources.\n# An error report file with more information is saved as:\n# /opt/test-runner/hs_err_pid54.log\ntwo-fer: done\n
```

I seem to recall us having a similar issue for the Java, Kotlin or Groovy test runner, so I'll look into it.

CC @artamonovkirill

**Comment by artamonovkirill** [comment]
https://github.com/adoptium/temurin-build/issues/2974 also suggests upgrading the Docker engine. Or is one used in exercise relatively new?

**Comment by ErikSchierboom** [comment]
@iHiD Do you know which version of the Docker runtime we are running?

**Comment by iHiD** [comment]
No, I don't. It was whatever was the [latest release][https://github.com/exercism/terraform/blob/main/terraform/tooling_invoker/ami.sh#L58) when we made the AMIs: . You could jump onto one of the tooling-invoker EC2 instances and take a look though.

**Comment by ErikSchierboom** [comment]
We're running docker version `20.10.8`
